src/coinbase/address/external_address.ts

```diff
@@ -237,11 +237,17 @@ export class ExternalAddress extends Address {
         response!.data.context.unstakeable_balance,
         assetId,
       ).amount,
-      claimableBalance: Balance.fromModelAndAssetId(
+    };
+
+    // For implementations that support claimable balance
+    if ("claimable_balance" in response!.data.context) {
```

Comment:
Hm, shouldn't this always return a value, even if a 0 value? 🤔

Reply:
After discussing internally, I have updated the `NativeEthStakingContext` to also include the `claimable_balance` field and set it to 0 for the case of native eth.

Since both native and partial now have the same 3 balances and we are on the same page that we want this for all protocols going forward we will have to simply the context api response and change it from a oneof b/w partial and native eth contexts to just a struct with the 3 balances.

This I believe will be a breaking change and I need to see how best to roll it out.